app/assets/javascripts/pageflow/ui/views/inputs/text_area_input_view.js

```diff
@@ -39,6 +39,15 @@ pageflow.TextAreaInputView = Backbone.Marionette.ItemView.extend({
       }
     });
 
+    if (this.options.disableRichtext) {
+      this.ui.toolbar.find('a[data-wysihtml5-command="bold"]').remove();
```

Comment:
Let's toggle a css class on the outer element and hide controls with `display: none` instead of removing them from the DOM.